libcontainer/configs/validate/rootless.go

```diff
@@ -65,9 +65,9 @@ func rootlessEUIDMount(config *configs.Config) error {
 				if _, err := config.HostUID(uid); err != nil {
 					return fmt.Errorf("cannot specify uid=%d mount option for rootless container: %w", uid, err)
 				}
+				continue
```

Comment:
This is added so when we've found `uid=...` in `opt`, we don't also try it for `gid=` but go to the next option.

Reply:
Yes, my original commit had `if` / `else if` but it results in a larger, harder to review commit, so I switched to `continue`.

Also note that `continue` (or `else`) is optional here, as missing it will not result is a bug, but merely in a very slight performance degradation.

Reply:
@thaJeztah ended up with one more commit changing this to if/else (and other minor nits).

Reply:
I think this callout explaining why continue is not a functional change belongs to the commit msg.